tests/test/linker-namespace-sanity/linker_namespace_sanity.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python2.7
+
+import os
+import sys
+import re
+
+sys.path.append(os.path.realpath(__file__ + '/../../../lib'))
+import udf
+
+from exatest.testcase import skipIf
+from udf import get_supported_languages
+
+from exatest.clients.odbc import getScriptLanguagesFromArgs
+import linker_namespace_base_test
+
+class LinkerNamespaceSanityTest(linker_namespace_base_test.LinkerNamespaceBaseTest):
+
+    def setUp(self):
+        self.query('CREATE SCHEMA FN2', ignore_errors=True)
+        self.query('OPEN SCHEMA FN2')
+
+    @skipIf('PYTHON3' not in get_supported_languages(), "UDF does not support Python3")
+    def test_linker_namespace(self):
+
+        lang = getScriptLanguagesFromArgs()
+        r_py2 = re.compile(r"exaudf/exaudfclient\b") # Match "...exaudf/exaudfclient ..."/"...exaudf/exaudfclient" but not "...exaudf/exaudfclient_py3"
+        lang_static = r_py2.sub("exaudf/exaudfclient_py2_static", lang)
+        r_py3 = re.compile(r"exaudf/exaudfclient_py3\b")
+        lang_static = r_py3.sub("exaudf/exaudfclient_py3_static", lang_static)
+        alter_session_query_str = "ALTER SESSION SET SCRIPT_LANGUAGES='%s'" % lang_static
+        print(alter_session_query_str)
```

Comment:
maybe, move into own function and give it a proper name

Reply:
Done